docuilib/docusaurus.config.js

```diff
@@ -18,6 +18,7 @@ const darkCodeTheme = themes.dracula;
     trailingSlash: false,
     customFields: {
       docsMainEntry: 'getting-started/setup',
+      docsDevelopmentMainEntry: 'next/getting-started/setup',
```

Comment:
since we already have `docsMainEntry`, no reason to have a duplicate 
instead you can have something like `docsDevelopmentVersionPrefix` (or something similar) and only have `next` to prefix the `docsMainEntry` field
WDYT?